configs/katello/nightly.yaml

```diff
@@ -65,7 +65,6 @@
       - flat-ruby27-el8
       - test-flat-el8
       - centos8-devel
-      - puppetlabs-puppet6-rhel-8
```

Comment:
I assume this is removed since it doesn't actually need Puppet, right?

Reply:
Yea, I cannot see a reason it would need it.